thakichowdhury/document-templating
# Update user interface

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Semantic version of the package has been incremented if needed (for bug fixes/features)
- [ ] Build (`yarn run build`) was run locally and any changes were pushed
- [ ] Lint (`yarn run lint`) has passed locally and any fixes were made for failures


## Pull request type

 

 

Please check the type of change your PR introduces:


## What is the current behavior?

- There's a very minimal user interface with the program
- You kind of have to know how the program works in order to use it

## What is the new behavior?

- There are colored prompts
- There's a preview of the template with the template variables as well as the default values highlighted

## Does this introduce a breaking change?



## Testing Instructions

- run `python3 main.py` in the root
- select a template using the associated numeric identifier
- see that there's a preview of the template with the template variables and default values highlighted

## Other information
